### ISSUE TITLE
[Bug]: Google apps can't retain login state.

### ISSUE THREAD

**Issue by electr1fy0** [PlayCover/PlayCover#903]
## Steps to reproduce
- Launch Google Tasks
- Login
- Quit the app
- Launch again to see it ask for login again

## Expected behaviour
Login should be preserved if possible.

## PlayCover version
2.0.5 and 3.0.0 both reproduce this behaviour.

## macOS version
Ventura 13.3

**Comment by electr1fy0** [comment]
From what I see, it looks this is a think with every Google app.

**Comment by ohaiibuzzle** [comment]
> From what I see, it looks this is a think with every Google app.

Yeah something I have been working on with PlayChain. The main issue is their usage of the Keychain APIs to store everything under the sun that I have not been able to fully capture

**Comment by electr1fy0** [comment]
So do we still have a chance?
It will be huge for me to have native Google apps for my mac as I use an Android device and web apps just suck.

**Comment by ohaiibuzzle** [comment]
I mean I know that Google apps do work with SIP off (aka using the system keychain)
As for when SIP is on we probably need some more time to either:
- Improving PlayChain to the point where it can actually mimics Apple's system keychain 
- Cheating the game and just make a service that must runs alongside apps for them to work (baaaaaaaaaaaaad)

**Comment by jimmywarting** [comment]
would be nice if we didn't have to turn of SIP. as that brings a lot of other complication with other apps and always has to be re-enabled after being done playing the game

**Comment by ohaiibuzzle** [comment]
@jimmywarting @electr1fy0 Update. I finally have time to track down what the heck it is. Appears to be some handling in keychain activities that ends up in PlayCover returning `SecKeyNotFound` for keys that actually exists

**Comment by combbbb** [comment]
any update on a possible fix?

**Comment by martinnnnnnnnnnnnnnnnnnn** [comment]
will personally venmo 10 dollars to whoever pushes the fix for this (along with youtube)

**Comment by 10a5er** [comment]
> will personally venmo 10 dollars to whoever pushes the fix for this (along with youtube)

i will add 10$, but with a more international payment system than venmo XD
